versions/ufo3/glyphs/glif.md

```diff
@@ -167,6 +167,18 @@ This element may occur any number of times.
 
 #### This element has no child elements.
 
+#### Anchor naming conventions
+
+The following are agreed upon standard naming conventions for the `name` of anchors for special uses.
+
+##### hcaret and vcaret
+
+Anchors with a `name` starting with `hcaret` or `vcaret` indicate horizontal or vertical position for the [GDEF Ligature Caret List table]. Anchors with the name `hcaret` provide the `x` value for a ligature caret, and the `y` value is ignored. Likewise, anchors with the name `ycaret` provide the `y` values for a ligature caret, and the `x` value is ignored.
```

Comment:
I would like the coupling to OpenType GDEF to be looser. In general UFO is export-format agnostic. Maybe mention GDEF caret stuff as an "for example".

Something like
"Anchors with a `name` starting with `hcaret` or `vcaret` indicate horizontal or vertical caret positions for ligature glyphs, for example to compile the OpenType GDEF Ligature Caret List table."